helm/cluster-api-provider-aws/templates/infrastructure-deployment.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: infrastructure-aws
```

Comment:
Can you add it into {{- include "labels.common" . | nindent 4 }} instead?

Reply:
Moved it into labels.common